ui-v2/app/components/consul-external-source/index.hbs

```diff
@@ -1,23 +1,17 @@
 {{#if item}}
-  {{#let (if _externalSource _externalSource (service/external-source item)) as |externalSource|}}
+  {{#let (service/external-source item) as |externalSource|}}
     {{#if externalSource}}
-      {{#if (has-block)}}
-        {{yield
-          (component 'consul-external-source' item=item _externalSource=externalSource)
-        }}
-      {{else}}
-        <span data-test-external-source={{externalSource}} class="consul-external-source {{externalSource}}">
-          {{#if label}}
-            {{label}}
+      <span data-test-external-source={{externalSource}} class="consul-external-source {{externalSource}}">
+        {{#if label}}
+          {{label}}
+        {{else}}
+          {{#if (eq externalSource 'aws')}}
+            Registered via {{uppercase externalSource}}
```

Comment:
We no longer need those recursive components we had, so I removed all the extra complexity here